server/world/world.go

```diff
@@ -210,6 +213,13 @@ func (w *World) HighestBlock(x, z int) int {
 	return int(v)
 }
 
+const LCGConstant = 1013904223
+
+func (w *World) GetUpdateLGC() int {
```

Comment:
Can we not just get a random int everytime? I get this probably matches vanilla but to what extent does that matter?

Reply:
We could do this, but I'm not entirely sure which calculations we should use otherwise, since I'm not that good at maths xP

Reply:
This is all that's left to do then. I'll check it now.

Reply:
An alternative to this is just a random number between int max and min, since it most likely underflows the integer since the equations are `add` and it caused the int to go into the negative. I might be wrong though. So there's the question, what would be better? If that one integer really minds you, then yes we could do that but this is closest to vanilla MC.

Reply:
Nevermind, figured it out.
LCG is like any other RNG, but it is known to be the best one out there. It doesn't have duplicates or whatever, so Mojang chose that. Go's random can obviously be used, but if that has duplicates then this is better.